src/main/java/com/brightpath/learnify/controller/NotesController.java

```diff
@@ -111,4 +112,16 @@ public ResponseEntity<List<NoteSummaryDto>> listNotes(UUID workspaceId) {
                 .map(dtoMapper::asNoteSummaryDto)
                 .toList());
     }
+
+    @Override
```

Comment:
We do not want to return this creatated page instead?

Reply:
We could, but I don't think it is necessary for now

Reply:
so when it will be?

Reply:
I would say that if thats not required now it won't be required at all 😅
We can extend the endpoint in the future for creating pages in the midde - then we could return the created page number